src/espIdf/partition-table/index.ts

```diff
@@ -0,0 +1,177 @@
+/*
+ * Project: ESP-IDF VSCode Extension
+ * File Created: Wednesday, 9th September 2020 1:19:26 pm
+ * Copyright 2020 Espressif Systems (Shanghai) CO LTD
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  WebviewPanel,
+  window,
+  ViewColumn,
+  Uri,
+  Disposable,
+  Webview,
+} from "vscode";
+import { join } from "path";
+import { ESP } from "../../config";
+import { readFileSync } from "../../utils";
+import { writeFile } from "fs-extra";
+import { Logger } from "../../logger/logger";
+import { file } from "tmp";
+
+export class PartitionTableEditorPanel {
+  private static instance: PartitionTableEditorPanel;
+
+  private readonly panel: WebviewPanel;
+  private disposable: Disposable[] = [];
+  private readonly extensionPath: string;
+
+  private filePath: string;
+
+  public static show(extensionPath: string, filePath: string) {
+    const column = window.activeTextEditor
+      ? window.activeTextEditor.viewColumn
+      : undefined;
```

Comment:
Can fallback to `vscode.ViewColumn.One` or any desired number.

Reply:
Shows in the current `viewColumn` if undefined... right?

Reply:
Oh I've noticed the default value later.